.vscode/settings.json

```diff
@@ -12,4 +12,5 @@
   "python.autoComplete.extraPaths": [
     "analysis/src"
   ],
+  "python.pythonPath": "C:\\tools\\Anaconda3\\python.exe",
```

Comment:
as mentioned before: it's a VSCode bug to auto insert your own computer's path into `settings.json`. Need to manually move it to your user settings file instead. (CTRL+SHIFT+P -> user settings -> python path)